client2/src/main/java/com/walmartlabs/concord/client2/impl/ResponseBodyHandler.java

```diff
@@ -77,7 +77,7 @@ public static <T> T handle(ObjectMapper objectMapper,
 
     private static boolean isJsonMime(String mime) {
         String jsonMime = "(?i)^(application/json|[^;/ \t]+/[^;/ \t]+[+]json)[ \t]*(;.*)?$";
-        return mime != null && (mime.matches(jsonMime) || mime.equals("*/*"));
+        return mime != null && (mime.matches(jsonMime) || "*/*".equals(mime));
```

Comment:
Useless. Non-null value here is already asserted.

Reply:
The purpose of this change is to be proactively defensive. Though it may be negligibly faster to run this way, it also conforms to a best practice and makes the code easier to read. Also, since the behavior is otherwise equivalent, there is no cost to this change.